src/client/client.ts

```diff
@@ -186,7 +186,7 @@ export class GraphQLTransportWSClient {
 
     const reqId = this.generateRequestId();
     this.requests[reqId] = { options, handler };
-    this.sendMessage(this.buildMessageForRequest(reqId, MessageTypes.GQL_START, options));
+    this.sendMessage(GraphQLTransportWSClient.buildMessage(reqId, MessageTypes.GQL_START, options));
```

Comment:
Maybe it's better to implement a SendMessageRaw (that receives a pure message) and a sendMessage that receives a reqId, type and payload. The main reason behind this is the need to send already built messages as is, for example in the reconnect stage. What do u think?